### ISSUE TITLE
No new line between last segment and the new segment

### ISSUE THREAD

**Issue by kariem2k** [kariem2k/ref_parsers#2]
Example:
ER  -TY  - JOUR

export.ris

**Comment by kariem2k** [comment]
This has been fixed by removing the closing tag, and then check if the remainder of the string is a starting tag. the first_line is passed to parse_entry method to use it instead of searching for a new one.

**Comment by kariem2k** [comment]
This could have been fixed with two other solutions:

1) A first pass that goes through the entire file text first and modifies it fixing all the issues. This is my preferred solution because the cleaning procedure will have information about the entire file making fixing the data more robust. The reason I did not implement that is that currently the file is loaded entirely in memory. The solution will be invalid if the need comes to stream the file rather than loading it entirely in memory.

2) Inject the line break into the lines array but that is might not be very efficient.
